Feat: Analytics new SKU (#7546)
Kong/docs.konghq.com: app/konnect/updates.md

from a single, cloud-based control plane, and provides a catalog of all deployed
services. [Try it today!](https://cloud.konghq.com/quick-start)

## August 2024
**API Requests**

> : All {{site.konnect_short_name}} Advanced Analytics users now access to a new feature that tracks [API requests](/konnect/analytics/api-requests/) in near real-time. API Requests provide detailed records for the requests that are made to your APIs. This information can not only help you understand your consumers better, but also simplifies any initial investigation into errors or performance issues by providing an intuitive web experience that is fully integrated into the rest of your {{site.konnect_short_name}} organization.

Reviewer: ```suggestion
: Announcing the general availability of [API Requests](/konnect/analytics/api-requests/). API Requests provide detailed records for the requests that are made to your APIs. This information can not only help you understand your consumers better, but also simplifies any initial investigation into errors or performance issues by providing an intuitive web experience that is fully integrated into the rest of your {{site.konnect_short_name}} organization.
```

from a single, cloud-based control plane, and provides a catalog of all deployed
services. [Try it today!](https://cloud.konghq.com/quick-start)

## July 2024

**API Requests**
: All {{site.konnect_short_name}} Advanced Analytics users now access to a new feature that tracks [API requests](/konnect/analytics/api-requests/) in near real-time. API Requests provide detailed records for the requests that are made to your APIs. This information can not only help you understand your consumers better, but also simplifies any initial investigation into errors or performance issues by providing an intuitive web experience that is fully integrated into the rest of your {{site.konnect_short_name}} organization.

**Manage control plane analytics data ingestion**

> : You can now opt-out from Analytics data ingestion for individual control planes. This allows you to not only control what data is important for Kong to keep, but also manage your overall spending on the Konnect Advanced Analytics app. If you disable analytics for a control plane, the data for that control plane will no longer display as part of the Analytics dashboard.

Author: ```suggestion
: You can now opt-out from Analytics data ingestion for individual control planes. This allows you to not only control what data is important for Kong to keep, but also manage your overall spending. If you disable analytics for a control plane, the data for that control plane will no longer display as part of the Analytics dashboard.
```

from a single, cloud-based control plane, and provides a catalog of all deployed
services. [Try it today!](https://cloud.konghq.com/quick-start)

## June 2024

**API Requests**

> : {{site.konnect_short_name}} users have now access to a new feature that tracks [API requests](/konnect/analytics/api-requests/) in near real-time. API Requests provide detailed records for the requests that are made to your APIs. This information can not only help you understand your consumers better, but also simplifies any initial investigation into errors or performance issues by providing an intuitive web experience that is fully integrated into the rest of your {{site.konnect_short_name}} organization.

Author: I changed it to say "all konnet users"

> **Manage control plane analytics data ingestion**

Reviewer 2: What happens if I'm using analytics before this feature goes live, do I immediately start getting charged for it? Or is there a pop up in the UI with a warning letting me know I'll be charged? 

If there is no UI warning, I think we might want to be more direct in the changelog (though I think I wrote this one) that it will now cost them money, so they should disable it if they don't want to be charged.

Author: Consumption based users will be billed but they are already billed for this feature. 
Existing subscription users no
New customers yes.

from a single, cloud-based control plane, and provides a catalog of all deployed
services. [Try it today!](https://cloud.konghq.com/quick-start)

## June 2024

**API Requests**
: {{site.konnect_short_name}} users have now access to a new feature that tracks [API requests](/konnect/analytics/api-requests/) in near real-time. API Requests provide detailed records for the requests that are made to your APIs. This information can not only help you understand your consumers better, but also simplifies any initial investigation into errors or performance issues by providing an intuitive web experience that is fully integrated into the rest of your {{site.konnect_short_name}} organization.

**Manage control plane analytics data ingestion**
: You can now opt-out from Analytics data ingestion for individual control planes. This allows you to not only control what data is important for Kong to keep, but also manage your overall spending on the Konnect Advanced Analytics app. If you disable analytics for a control plane, the data for that control plane will no longer display as part of the Analytics dashboard.

> : You can manage your control plane analytics data ingestion by editing an existing control plane and clicking the **Analytics** toggle. You can also disable Analytics data ingestion when creating a new control plane.

Reviewer: ```suggestion
: You can manage your control plane analytics data ingestion by editing an existing control plane and clicking the **Advanced Analytics** toggle.
```

> : ![control plane without analytics](/assets/images/products/konnect/changelog/konnect-control-plane-without-analytics.png)

Reviewer: Remove.

> : ![control plane without analytics](/assets/images/products/konnect/changelog/konnect-control-plane-without-analytics.png)

Author: ```suggestion
```
